backend/app/adapter/sqldb/githubsso_integration_test.go

```diff
@@ -106,6 +153,18 @@ func TestGithubSSOSql_CreateMapping(t *testing.T) {
 		},
 		{
 			name: "neither SSO user ID nor Short user ID exists",
+			userTableRows: []userTableRow{
+				{
+					id:    "short",
+					email: "short@gmail.com",
+					name:  "short",
+				},
+				{
+					id:    "alpha",
```

Comment:
if these rows are used many times maybe it would be better to have them easier to reuse.
```Go
defaultUserTableRows := []userTableRow{
				{
					id:    "short",
					email: "short@gmail.com",
					name:  "short",
				},
				{
					id:    "alpha",
					email: "alpha@gmail.com",
					name:  "alpha",
				},
			}
```